src/app/(with-header)/mypage/dashboard/components/ReservationDashboardCalendar.tsx

```
@@ -0,0 +1,142 @@
+                );
+              }
+
+              return (
+                <button
+                  key={`${weekIndex}-${dayIndex}`}
+                  onClick={() => onDateClick(dateString)}
+                  className={`relative flex h-105 flex-col items-start justify-start p-8 transition-colors ${borderClass} cursor-pointer text-gray-900 hover:bg-gray-50 ${isToday ? 'bg-blue-50' : 'bg-white'} `}
+                >
+                  <div className='flex items-center gap-4'>
+                    <span
+                      className={`text-md font-medium ${isToday ? 'font-bold text-blue-600' : 'text-gray-900'} `}
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)